bin/atnodes

```diff
@@ -170,6 +179,7 @@ while (1) {
     while ($active_count < $concurrency) {
         last if !@hosts;
         my $host = shift @hosts;
+        my $login_with_password = (defined $password) && (!$ask_for_pass_only_for_sudo);
```

Comment:
Style: we don't need the last pair of parentheses as well. `!` clearly has a higher priority than `&&`, hence no possible ambiguity here.